src/connectors/infinite-hits/__tests__/connectInfiniteHits-test.ts

```diff
@@ -9,7 +9,7 @@ import connectInfiniteHits from '../connectInfiniteHits';
 import { Client } from '../../../types';
 
 jest.mock('../../../lib/utils/hits-absolute-position', () => ({
-  addAbsolutePosition: hits => hits,
+  addAbsolutePosition: hits => hits.map(x => x),
```

Comment:
We should add a comment explaining why we map because out of context, it doesn't make much sense (→ to create a new array instance).